pkgs/by-name/taldir/package.nix

```
@@ -6,19 +6,19 @@
 }:
 buildGoModule {
   pname = "taldir";
-  version = "0-unstable-2024-02-18";
+  version = "0-unstable-2025-01-25";
 
   src = fetchgit {
     url = "https://git.taler.net/taldir.git";
-    rev = "9c5230f64d16d46c000c6d4f5842170c51b697ce";
-    hash = "sha256-Ri0R7kxP/FitBRKtrM48Cbks63mqpXsRc3r98M0sMus=";
+    tag = "v1.0.3";
```

We can reuse the version attribute here since it nicely corresponds to the upstream tag, unlike the previous unstable version:

```suggestion
    tag = "v{finalAttrs.version}";
```

What's this `finalAttrs` you say? It's a way for us to reuse attributes from within the derivation (see my comment above).